Makefile

```diff
@@ -437,6 +427,16 @@ else
   $(error Unable to detect a suitable MIPS toolchain installed)
 endif
 
+# TEXT ENGINES
+#   s2dex_text_engine - Text Engine by someone2639
+TEXT_ENGINE := s2dex_text_engine
```

Comment:
this shouldn't be on by default, and I think it's alright to put this var specifically towards the top of the makefile so it's accessible still

Reply:
Woops, accidentally left that in!

Implemented your suggestion + added argument checking with validate-option and fixed an issue where s2d_engine weren't added to LINK_LIBRARIES